code/game/machinery/spaceheater.dm

```diff
@@ -186,9 +186,11 @@
 	if(!can_interact(user))
 		return
 	on = !on
+	mode = HEATER_MODE_STANDBY
+	usr.visible_message(span_notice("[usr] switches [on ? "on" : "off"] \the [src]."), span_notice("You switch [on ? "on" : "off"] \the [src]."))
```

Comment:
```suggestion
	user.visible_message(span_notice("[user] switches [on ? "on" : "off"] \the [src]."), span_notice("You switch [on ? "on" : "off"] \the [src]."))
```

Why not use user here instead?

Reply:
It wasn't a consideration at the time, but I am open to it.  I just tested it locally.  What are the benefits/functionality changes?